src/components/ProductLink/ProductLink.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { ProductLink } from "./ProductLink";
+
+export default {
+  title: "Example/ProductLink",
+  component: ProductLink,
+  argTypes: {
+    href: { control: "text", default: "" },
+    label: { control: "text", default: "" },
+    image: { control: "text", default: "" },
+    revealImage: { control: "text", default: "" },
+    price: { control: "number", default: "" },
+    compareAtPrice: { control: "number", default: "" },
+    unavailable: { control: "boolean", default: "false" },
+  },
+} as ComponentMeta<typeof ProductLink>;
+
+const Template: ComponentStory<typeof ProductLink> = (args) => (
+  <ProductLink {...args} />
+);
+
+// const handleHover = (e: React.MouseEvent<HTMLButtonElement>) => {
+//   console.log("Hover", e.target);
+// };
+
+export const Default = Template.bind({});
+Default.args = {
+  href: "/productLink",
+  label: "Grey Hoodie",
+  image: {
+    src: "https://www.forever21.com/dw/image/v2/BFKH_PRD/on/demandware.static/-/Sites-f21-master-catalog/default/dw3409b153/1_front_750/00425541-01.jpg?sw=1000&sh=1500",
```

Comment:
Good usage of dummy images but it's better to use an actual placeholder image service such as https://picsum.photos/. This avoids previews breaking if f21 updates/removes their image assets.

Reply:
Can also DRY code like so:
```
const placeholderImage = {
  src: "https://picsum.photos/id/1234/200/300"
  alt: "placeholder image"
}
const placeholderRevealImage = {
  src: "https://picsum.photos/id/5678/200/300"
  alt: "placeholder reveal image"
}
...

Default.args = {
  image: placeholderImage,
  revealImage: placeholderRevealImage,
}
```